package.json

```diff
@@ -146,6 +147,7 @@
     "fixturify": "^2.1.0",
     "latest-version": "^5.1.0",
     "mocha": "^7.2.0",
+    "is-language-code": "^1.0.7",
```

Comment:
This should not be both in `dependencies` and `devDependencies`. Should probably remove this one.

```suggestion
```

Reply:
Can do -- thanks!